src/Uno.UI/UI/Xaml/UIElement.Pointers.Android.cs

```diff
@@ -0,0 +1,138 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using Windows.Devices.Input;
+using Windows.UI.Input;
+using Windows.UI.Xaml.Input;
+using Android.Runtime;
+using Android.Views;
+using Microsoft.Extensions.Logging;
+using Uno.Extensions;
+using Uno.Logging;
+
+namespace Windows.UI.Xaml
+{
+	partial class UIElement
+	{
+		private class TouchReRouter : Java.Lang.Object, View.IOnTouchListener
+		{
+			private readonly UIElement _target;
+
+			public TouchReRouter(IntPtr handle, JniHandleOwnership transfer)
+				: base(handle, transfer)
+			{
+			}
+
+			private TouchReRouter(UIElement target)
+				: base(IntPtr.Zero, JniHandleOwnership.DoNotRegister)
+			{
+				_target = target;
+			}
+
+			/// <inheritdoc />
+			public bool OnTouch(View view, MotionEvent nativeEvent)
+				=> _target.OnNativeMotionEvent(nativeEvent, view, true);
+		}
+
+
+		partial void AddPointerHandler(RoutedEvent routedEvent, int handlersCount, object handler, bool handledEventsToo)
+		{
+			if (handlersCount == 1)
+			{
+				IsNativeMotionEventsEnabled = true;
+			}
+		}
+
+		partial void RemovePointerHandler(RoutedEvent routedEvent, int remainingHandlersCount, object handler)
+		{
+			if (remainingHandlersCount == 0)
+			{
+				// TODO: Disable pointer events reporting
```

Comment:
Same is there an Issue/Item linked for this TODO ?

Reply:
Here we are : https://github.com/unoplatform/uno/issues/1806